src/components/Calendar.tsx

```diff
@@ -29,13 +29,20 @@ const { DOT_SIZE, MAX_DOTS_PER_DAY } = CALENDAR;
 type Props = {
   isListRefreshing: boolean;
   query: string;
-  queryVariables?: { dateRange?: string[]; contentContainerId?: number };
+  queryVariables: { dateRange?: string[]; contentContainerId?: number };
   navigation: StackNavigationProp<any>;
+  additionalData?: any;
 };
```

Comment:
I see that the alphabetical sorting is also missing here.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/940/commits/f653378eefcca79cef1def80dfa038e995b1bccf